src/main/java/org/jose/F.java

```diff
@@ -4,11 +4,16 @@
     public static void main(String[] args) {
 
         int h = 0;
+        int j = 0;
```

Comment:
choose meaningful names to make your code understandable to readers. how am I supposed to know what this variable is for when reading the code?